anvio/docs/programs/anvi-display-functions.md

```diff
@@ -0,0 +1,95 @@
+
+### Basic Run 
+
+The simplest way to run this program is as follows:
+
+{{ codestart }}
+anvi-display-functions -e %(external-genomes)s \
+                       --annotation-source KOfam \
+                       --profile-db KOFAM-PROFILE.db
+{{ codestop }}
+
+
+You can replace the annotation source based on what is available across your genomes. You can use the program %(anvi-db-info)s to see all available function annotation sources in a given %(contigs-db)s or %(genomes-storage-db)s. Please see %(functions)s for more information on functions and how to obtain them.
+
+
+{:.notice}
+Please note that a %(profile-db)s will be automatically generated for you. Once it is generated, the same profile database can be visualized over and over again using %(anvi-interactive)s in manual mode, without having to retain any other files.
+
+### Aggregating functions using accession IDs
+
+Once it is run, this program essentially aggregate all function names that occur in one or more genomes in the set of genomes found in input sources. The user can ask the program to use accession IDs to aggregate functions rather than function names:
+
+{{ codestart }}
+anvi-display-functions -e %(external-genomes)s \
+                       --annotation-source KOfam \
+                       --profile-db KOFAM-PROFILE.db \
+                       --aggregate-based-on-accession
+{{ codestop }}
+
+While the default setting which is to use function names will be appropriate for most applications, using accession IDs instead of function names may be important for specific applications. There may be an actual difference between using functions or accession to aggregate data since multiple accession IDs in various databases may correspond to the same function. This may lead to misleading enrichment analyses downstream as identical function annotations may be over-split into multiple groups. Thus, the default aggregation method uses function names.
+
+### Aggregating functions using accession IDs
+
+In some cases a gene may be annotated with multiple functions. This is a decision often made at the function annotation tool level. For instance %(anvi-run-ncbi-cogs)s may yield two COG annotations for a single gene because the significance score for both hits may exceed the default cutoff. While this can be useful in %(anvi-summarize)s output where things should be most comprehensive, having some genes annotated with multiple functions and others with one function may over-split them (since in this scenario a gene with COGXXX and COGXXX;COGYYY would end up in different bins). Thus, %(anvi-display-functions)s will will use the best hit for any gene that has multiple hits. But this behavior can be turned off the following way:
+
+{{ codestart }}
+anvi-display-functions -e %(external-genomes)s \
+                       --annotation-source KOfam \
+                       --profile-db KOFAM-PROFILE.db \
+                       --aggregate-using-best-hit
+{{ codestop }}
+
```

Comment:
I am a bit confused here :) It sounds like `anvi-display-functions` aggregates using best hit by default, and that there is a flag to turn this off, but the flag `--aggregate-using-best-hit` sounds like it is turning this behavior on instead. What is the default?

Also I think the heading for this section is supposed to be something different :)

Reply:
I agree and I fixed that later. Our flag is now `--aggregate-using-all-hits` (by default it uses the best)

Reply:
Ah, great, I forgot to update the md file, of course :)